lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,18 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    text = text.replace("º", "")
+    cleaned_text = ""
+    for symbol in text:
+        if symbol.isalpha():
+            cleaned_text += symbol
```

Comment:
can you combine these for loops? you can append symbol in list permanently.

Reply:
i don't understand what you mean

Reply:
done